kafka/00version.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 /*
-#include <librdkafka/rdkafka.h>
+#include "rdkafka_select.h"
```

Comment:
The "rdkafka_" prefix is used by librdkafka, so to be future-proof it is better if we use something else in the Go client, maybe "select_rdkafka.h".